src/main/java/edu/northeastern/truthtree/adapter/collections/CollectionsMockAdapter.java

```diff
@@ -14,7 +14,8 @@ public JSONArray getCollections() {
 		JSONParser parser = new JSONParser();
 		JSONArray collections = new JSONArray();
 		try {
-			collections = (JSONArray) parser.parse(new FileReader("src/main/resources/Collections.json"));
+			collections = (JSONArray) parser.parse(
```

Comment:
Consider using the same JSON reader as the basicInfoAdapters

Reply:
I believe this was from Nena's commit. I can create a separate JSON Reader class (within the adaptor package) that can hold this function if you would like?